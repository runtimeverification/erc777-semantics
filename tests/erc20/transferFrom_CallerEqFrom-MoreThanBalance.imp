test.preamble();

// Testing transferFrom(From,To,Value) when From = caller
// and Value > BalanceFrom.

test.approve(caller, 71);
test.transferFrom(caller, 4, 71)  // balance of caller is 70
