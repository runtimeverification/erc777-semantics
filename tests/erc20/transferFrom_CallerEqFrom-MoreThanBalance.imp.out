7 is approving allowance 71 for 7
7 is transferring 71 from 7 to 4
REVERT
Balance of 7 is 70
Balance of 4 is 40
Allowance of 7 for 7 is 71
